README.md

```diff
@@ -121,6 +121,19 @@ jobs:
           check_all_files_with_shebangs: "false" # Optional.
 ```
 
+## Permissions
+
+It is recommended to add explicit [`permissions`](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/controlling-permissions-for-github_token)
+to the workflow file so the scope of the `GITHUB_TOKEN` passed to `action-shellcheck` is as narrow as
+possible. The required permissions for `action-shellcheck` to read the contents of a PR and post review
+comments to it, is:
+
+```yaml
+permissions:
+  contents: read
+  pull-requests: write
```

Comment:
do you mean with more permissions this:
https://docs.github.com/de/actions/reference/workflows-and-actions/workflow-syntax#permissions


> Work with check runs and check suites. For example, `checks: write` permits an action to create a check run. For more information, see Permissions required for GitHub Apps.



```suggestion
permissions:
  contents: read
  pull-requests: write
  checks: write
```

Reply:
I don't know, but hopefully @haya14busa does? :)

Reply:
yes, checks: write should be correct for the github-check reporter